### ISSUE TITLE
Addition of "unique trips" or "unique routes" to stop statistics

### ISSUE THREAD

**Issue by kyletitlowdot** [mrcagney/gtfs_kit#4]
For the functions that create stop and route statistics, would it be possible to have a summary metric that counts the number of "unique" trips or routes over a given time period? 

It'd be great to know, for example, the number of unique routes represented in the trips stopping at a given location, especially per day or per week (e.g., X stop is served by 36 trips on 4 routes). Right now, summing the routes over that time period may count that same route again and again (e.g., X stop was served by the same route every day in a week but it shows up as "7 routes" when summed). 

This package is so helpful! Thank you for the work maintaining this resource.

**Comment by araichev** [comment]
Thanks for your feedback, @kyletitlowdot .
Counting the number of unique trips or routes serving a stop across several days sounds like a strange hence niche task and would not fit well into the `compute_stop_stats()` function, which is designed for daily stats.
As such, i recommend writing a custom function for your task.

Maybe one that takes in a start date, start time, end date, end time then counts unique trips and routes across each stop in that time interval.
Basically gather all the stop time data frames in the date range, subset the first and last one to the start and end times, concatenate the data frames, group by stop ID, then aggregate trip IDs and route IDs using Pandas's `nunique()` function. 

Does that make sense?
